src/components/Editor.tsx

```diff
@@ -1,28 +1,101 @@
-import React from 'react';
+import React from 'react'
 import '../styles/Editor.scss'
-import {IInfographicsSection} from '../types/TInterfaceSection';
-import Button from './Button';
+import Button from './Button'
+import {IInfographicsDetails} from '../types/IInfographicsDetails'
+import {IInfographicsSection} from '../types/IInfographicsSection'
 
 interface IProps {
-    infographicsSections: IInfographicsSection[],
+    infographicsDetails: IInfographicsDetails,
+    selectedSection: IInfographicsSection | null,
     addSection: () => void,
+    divideSection: (section: IInfographicsSection) => void,
+    setWidth: (width: number) => void,
```

Comment:
z `setWidth` a `setHeight` nechapu co to ma delat, ceho vysku/sirku to ma nastavovat